cli/src/test/java/org/jboss/pnc/bacon/cli/AppTest.java

```diff
@@ -76,90 +85,88 @@ void testJSONDisabled() throws Exception {
+            String text = tapSystemOut(
+                    () -> assertThat(
+                            app.run(
+                                    new String[] {
+                                            "--verbose",
+                                            "pnc",
+                                            "--profile",
+                                            "foobar",
+                                            "admin",
+                                            "maintenance-mode",
+                                            "activate",
+                                            "-h" })).isZero());
+
+            assertThat(text).isEqualTo(
+                    "Usage: bacon pnc admin maintenance-mode activate [-hov]" + System.lineSeparator()
+                            + "       [-p=<configurationFileLocation>] [--profile=<profile>] <reason>"
+                            + System.lineSeparator() + "This will disable any new builds from being accepted"
+                            + System.lineSeparator() + "      <reason>              Reason" + System.lineSeparator()
+                            + "  -h, --help                display this help message" + System.lineSeparator()
+                            + "  -o                        use json for output (default to yaml)"
+                            + System.lineSeparator() + "  -p, --configPath=<configurationFileLocation>"
+                            + System.lineSeparator() + "                            Path to PNC configuration folder"
+                            + System.lineSeparator() + "      --profile=<profile>   PNC Configuration profile"
+                            + System.lineSeparator() + "  -v, --verbose             Verbose output"
+                            + System.lineSeparator() + System.lineSeparator() + "Example:" + System.lineSeparator()
+                            + "$ bacon pnc admin maintenance-mode activate \"Switching to maintenance mode for"
+                            + System.lineSeparator() + "upcoming migration\"" + System.lineSeparator());
+
+            String text2 = tapSystemErr(
+                    () -> assertThat(
+                            app.run(
+                                    new String[] {
+                                            "--configPath",
+                                            configYaml.toString(),
+                                            "--verbose",
+                                            "pnc",
+                                            "--profile",
+                                            "foobar",
+                                            "admin",
+                                            "maintenance-mode",
+                                            "activate",
+                                            "-h" })).isZero());
+            assertThat(text2).containsOnlyOnce("Config file set from flag with profile foobar to");
+        });
```

Comment:
I am unclear as to why you are changing these tests. They worked fine in my local laptop, and on the CI tests. I am not convinced about adding an extra library and on the whole I do not see the reason for this PR.